controls/radmap/how-to/howto-wraparound-virtualization-visualizationlayer.md

```diff
@@ -0,0 +1,383 @@
+
+In the next part of this article, we will display points of interest (POIs) over the cities (Berlin, London, Amsterdam, Madrid, Barcelona, Rome) in the map in MVVM scenario and set the virtualization when the wraparound mode of the map is on. In __Example 1__ we will specify the business object which will hold the Longitude, Latitude values, the name of the cities and their location.
+
+#### __[C#] Example 1:  Business object__
+{{region c#-radmap-radmap-howto-wraparound-virtualization-visualizationlayer_0}}
+	public class CityLocation : ViewModelBase
+    {
+        private double _lattitude;
+        private double _longitude;
+        private string _cityName;
+        public CityLocation(double latitude, double longitude, string cityName)
+        {
+            this.Latitude = latitude;
+            this.Longitude = longitude;
+            this.CityName = cityName;
+        }
+
+        public double Latitude
+        {
+            get
+            {
+                return this._lattitude;
+            }
+            set
+            {
+                Location old = this.Location;
+                this._lattitude = value;
+                this.OnPropertyChanged("Latitude");
+                this.OnPropertyChanged("Location");
+            }
+        }
+
+        public double Longitude
+        {
+            get
+            {
+                return this._longitude;
+            }
+            set
+            {
+                Location old = this.Location;
+
+                this._longitude = value;
+                this.OnPropertyChanged("Longitude");
+                this.OnPropertyChanged("Location");
+            }
+        }
+
+        public Location Location
+        {
+            get
+            {
+                return new Location(this.Latitude, this.Longitude);
+            }
+        }
+
+        public string CityName
+        {
+            get
+            {
+                return this._cityName;
+            }
+            set
+            {
+                this._cityName = value;
+                this.OnPropertyChanged("CityName");
+            }
+        }
+    }
+{{endregion}}
+
+Then you can create custom class which implement the __IMapItemsVirtualizationSource__ interface. In the handler of the __MapItemRequest()__ event you can get the __LocationRect__ region from the arguments and normalize this region by using GetCoercedLocation method of RadMap. Then you need to calculate which cities should be displayed in this normalized region.
```

Comment:
"Then you need to calculate which cities should be displayed in this normalized region."

I noticed that you often use a sentence construction as the following:  "You can get the LocationRect from the argument and normalize this region." and "Then they should be displayed in this normilized region". I don't know if this is correct. To me it doesn't sound very good, but I suggest to discuss this with someone that has more extended knowledge in the English language maybe Tanya or Yana. 

To me using "the" instead of "this" is better. For example: "You can get the LocationRect from the argument and normalize the region." and "Then they should be displayed in the normalized region".